### DISCUSSION TITLE
Add a "clear" button for the Text Box variable

### DISCUSSION THREAD

**Discussion by mugen6** [grafana/grafana#51676] [Dashboards, Panels & Visualizations]
We use the Text Box variable to filter the dashboard.

![image](https://user-images.githubusercontent.com/23708354/177002402-f757fe7b-fd8c-41e6-b364-49f85856fecc.png)

But to clear this filter, you have to:

1. click back into the text box
2. highlight the text 
3. press delete
4. click somewhere else on the dashboard

It would be helpful to have a simply "clear this field" button inside the text box, like this:

![image](https://user-images.githubusercontent.com/23708354/177002524-851113cd-5a4f-4201-a929-f0da4f2e6be5.png)

So it can be cleared in 1 click instead.

**Comment by Goateater94** [comment]
There's no update for this topic? This could be a very useful feature!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
